apps/condo/domains/meter/utils/clientSchema/Renders.tsx

```diff
@@ -22,3 +25,20 @@ export const getUnitRender = (intl, search: FilterValue) => {
         return getTableCellRenderer(search, true, null, null, POSTFIX_PROPS, extraTitle)(unitName)
     }
 }
+
+export const getResourceRender = (intl, search?: FilterValue | string) => {
+    return function render (text, meterReading): RenderReturnType {
+        const value = get(meterReading, ['meter', 'resource', 'name'])
+        const isAutomatic = get(meterReading, ['meter', 'isAutomatic'], false)
+        const isExternalSource = Boolean(get(meterReading, ['source', 'type']) === METER_READING_SOURCE_EXTERNAL_IMPORT_TYPE)
+        const AutoMessage = intl.formatMessage({ id: 'pages.condo.meter.AutoPrefix' })
```

Comment:
May this line be moved to outer closure? Looks like it is constant value. So it may be saved to parent function context

Reply:
`intl` is passed to the outer function. All similar functions are implemented in this way. It will be strange if the implementation of this function differs from similar.

Thus, we can change the implementation of the function from one place